### PR TITLE
Update Google Chrome SSO documentation

### PR DESCRIPTION
**Pull request by sgallagher**
On Linux systems, the whitelist can be specified in a
configuration file on the disk.

### CODE REVIEW

File: doc/guide/sso.xml [hunk 1]
```diff
@@ -81,8 +81,17 @@ Password for user@EXAMPLE.COM:
       </varlistentry>
       <varlistentry>
         <term>Google Chrome</term>
-        <listitem><para>Exit your browser completely, and start it with a command line
-          like this: <code>google-chrome --auth-server-whitelist=*example.com</code></para></listitem>
+        <listitem><para>
+          On Linux: create the file /etc/opt/chrome/policies/managed/example-com.json with the contents:
```
Comment (stefwalter) [inline doc/guide/sso.xml]: Need &lt;code&gt; tags around the filename.

File: doc/guide/sso.xml [hunk 2]
```diff
@@ -101,8 +110,18 @@ Password for user@EXAMPLE.COM:
       </varlistentry>
       <varlistentry>
         <term>Google Chrome</term>
-        <listitem><para>Exit your browser completely, and start it with a command line
-          like this: <code>google-chrome --auth-server-whitelist=*example.com --auth-server-delegate-whitelist=*example.com</code></para></listitem>
+        <listitem><para>
+          On Linux: create the file /etc/opt/chrome/policies/managed/example-com.json with the contents:
```
Comment (stefwalter) [inline doc/guide/sso.xml]: Ditto.

**Comment by stefwalter** [comment]
Thanks! Just some minor nits.

**Comment by mvollmer** [comment]
Upps, typo in `Closes: ...`

**Comment by sgallagher** [comment]
Marius, can you explain what typo you see?

**Comment by sgallagher** [comment]
Ah, on further examination I see that this was closed instead of another ticket and that this hasn't been pushed yet. Roger.

**Comment by mvollmer** [comment]
> Ah, on further examination I see that this was closed instead of another ticket and that this hasn't been pushed yet. Roger.

Yes, sorry for being terse.  I wanted to close #2054, but typed #2045 by mistake.

**Pull request closed without merging**